packages/blockchain-extension/extension/commands/environmentConnectCommand.ts

```diff
@@ -119,8 +119,19 @@ export async function fabricEnvironmentConnect(fabricEnvironmentRegistryEntry: F
             environmentData = 'user environment';
         }
 
+        let envType: string;
+        if (fabricEnvironmentRegistryEntry.environmentType === EnvironmentType.ENVIRONMENT) {
```

Comment:
Could you comment in `FabricEnvironmentRegistryEntry.ts` (where we've defined the EnvironmentType enum), to state that if we update the enum, we need to update this part here as well, so it gets reported!

Reply:
Done! 👍